.github/workflows/reusable-linters.yml

```diff
@@ -55,7 +55,7 @@ jobs:
       run: |
         make lint
     - name: Send coverage data to Codecov
-      uses: codecov/codecov-action@v3
+      uses: codecov/codecov-action@v4
       with:
         token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
This got merged too soon. Something like https://github.com/aio-libs/multidict/commit/74840e8fdf84e4617054f3fe6581a5ec5e507554 needs to be implemented for v4 to actually work on pushes.

Reply:
Fixed in https://github.com/aio-libs/yarl/commit/5b71d250390184a184c4e61a4a723a7228218bd0